.github/workflows/push.yml

```diff
@@ -1,5 +1,7 @@
 name: GitHub Actions
 on: [ push ]
+concurrency: ci-${{ github.ref }}
```

Comment:
Pochopil bych na cely repo, protoze je nejaka vazba na globlani resource ala projekt a nesmi bezet vic pipelines najednou. Ale proc na branch? :thinking:

Reply:
jojo, mě to tak nějak trklo když jsem si poslal víc pushů do branche, ale jop, je to globální resource